src/components/CMakeLists.txt

```diff
@@ -48,14 +51,15 @@ else()
   target_include_directories(
     ${PROJECT_NAME}
     INTERFACE "${COMPONENTS_DIR}/include/policy/policy_regular")
+  set(POLICY_PATH ${COMPONENTS_DIR}/policy/policy_regular/)
```

Comment:
@BSolonenko What if policy flow set - HTTP?

Reply:
@BSolonenko @AByzhynar  HTTP policy mode is in policy regular path